cmd/query/app/server.go

```diff
@@ -161,3 +166,24 @@ func (s *Server) Close() {
 	s.httpServer.Close()
 	s.conn.Close()
 }
+
+// stringSliceAsHeader parses a slice of strings and returns a http.Header.
+//  Each string in the slice is expected to be in the format "key: value"
+func stringSliceAsHeader(slice []string, logger *zap.Logger) http.Header {
```

Comment:
I think this method should be somewhere in the configuration package that defines the CLI flags. This is more of a "business" functionality that should get the data in the already-parsed format.

Reply:
Moved to `flags.go` and changed the type on QueryOptions from `[]string` to `http.Header`.  If this is not what you had in mind let me know!